lib/capybara/cuprite/javascripts/index.js

```diff
@@ -180,8 +180,7 @@ class Cuprite {
   }
 
   input(node) {
-    let event = document.createEvent("HTMLEvents");
-    event.initEvent("input", true, false);
+    let event = new InputEvent("input", { inputType: "insertText" });
```

Comment:
Muss man da noch `bubbles: true,  cancelable: false,` setzen oder ist das der Default bzw. war eh falsch? 

Scheinbar sind das ja die hinteren beiden Parameter bei [initEvent](https://developer.mozilla.org/en-US/docs/Web/API/Event/initEvent)

Reply:
Ich war mir sicher, dass sind die Defaults aber scheinbar nicht ganz ... bubbles wäre "false".

Das echte Event sendet aber tatsächlich bubbles: true :(
c2ad6f6ec68002c7aa72e5ba1307188fc56b0593 (hab jetzt beides explizit gesetzt)

Reply:
true fände ich auch cooler, wenn das der Default ist. Würde das anderes Cuprite Zeug kaputt machen, wenn das Input-Event gecancelt wird?
Ich hab die Git-History von dem File angeschaut, aber konnte jetzt nicht rausfinden warum genau dieses Event als nicht cancelbar markiert wurde. Die restlichen Events dürfen scheinbar alle gecancelt werden.

Reply:
Glaub das ging hier ganz durcheinander.

false ist der Default beim _initialisieren_ eines selbst erzeugten Events.

Der Browser sendet bei bubbles aber true in einem echten Input-Event, daher hab ich das hier auf true korrigiert. cancelable müsste ich gar nicht angeben, hab ich nur weil expliziter. Das ist aber immer false, sowohl Default als auch was in diesem Fall tatsächlich vom Browser kommt.

Würd mich hier ganz nah am Browser halten.